Sources/ApplicativeRouter/Combinators.swift

```diff
@@ -13,8 +14,8 @@ public func lit(_ str: String) -> Router<Prelude.Unit> {
             : nil
       }
   },
-    print: { _ in .init(method: nil, path: [str], query: [:], body: nil) },
-    template: { _ in .init(method: nil, path: [str], query: [:], body: nil) }
+    print: { _ in .init(method: nil, path: [str], query: nil, body: nil) },
```

Comment:
First thing you may notice is that `query` is no longer `[String: String]`. We do less parsing here (so that `UrlFormEncoding` can do it for us) and leave `query` as `String?`.